exchange/exchange_test.go

```diff
@@ -180,59 +181,68 @@ func TestDebugBehaviour(t *testing.T) {
 	}
 
 	type aTest struct {
-		desc      string
-		in        inTest
-		out       outTest
-		debugData debugData
+		desc            string
+		in              inTest
+		out             outTest
+		debugData       debugData
+		generalWarnings bool
```

Comment:
Should this be `generateWarnings`? Looks like it will add a warning if true.

Reply:
Renamed